services/sensorhub-service-mqtt-hivemq/build.gradle

```diff
@@ -12,6 +12,25 @@ dependencies {
     exclude group: 'ch.qos.logback', module: 'logback-classic'
     exclude group: 'com.google.guava', module: 'guava'
     exclude group: 'com.hivemq', module: 'hivemq-extension-sdk'
+    exclude group: 'io.dropwizard.metrics'
```

Comment:
I don't think the MQTT service will work properly if we exclude all that!

Reply:
Exclude already existed into source branch: https://github.com/opensensorhub/osh-addons/blob/v2/services/sensorhub-service-mqtt-hivemq/build.gradle#L14

To be sure what we need, we should install,start and use to test